app/views/includes/dateFields.scala.html

```diff
@@ -112,7 +112,7 @@
             fieldset = Fieldset(
                 legend = Legend(
                     content = HtmlContent(labelText),
-                    classes = "govuk-!-font-weight-bold"
+                    classes = s"${customClass.getOrElse("")} govuk-!-font-weight-bold"
```

Comment:
I believe if we have custom class then we don't need govuk-!-font-weight-bold as it can override some property.
So I think it is better to have here:
```scala
classes = s"${customClass.getOrElse("govuk-!-font-weight-bold")}"
```

At the moment we have unnecessary 3rd class.
```html
<legend class="govuk-fieldset__legend govuk-fieldset__legend--m govuk-!-font-weight-bold">
```